.github/workflows/project-tests.yml

```diff
@@ -46,8 +50,25 @@ jobs:
         dotnet restore DfE.FindInformationAcademiesTrusts
         dotnet build DfE.FindInformationAcademiesTrusts -c Release
 
-    - name: Run mutation tests
-      run: dotnet stryker
-
     - name: Ensure there are no pending migrations
       run: dotnet ef migrations has-pending-model-changes --context FiatDbContext --project DfE.FindInformationAcademiesTrusts.Data.FiatDb --startup-project DfE.FindInformationAcademiesTrusts
+
+    - name: Run mutation tests
+      run: dotnet stryker --reporter "markdown"
+
+    - name: Print stryker report
+      working-directory: StrykerOutput
+      run: |
+        # Find the latest directory matching the date-time format
+        LATEST_DIR=$(ls -d $(date +'%Y-%m-%d').* | sort -r | head -n 1)
```

Comment:
We can simplify this by setting the `--output` flag on stryker to a fixed directory